src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  let marriedMessage = 'I am not married';
+
+  if (isMarried) {
```

Comment:
You can also create a function that will check if the person is married or not, if not, then return "I am not married", otherwise the person's gender is checked